mockgcp/mockcertificatemanager/service.go

```diff
@@ -57,11 +58,23 @@ func (s *MockService) Register(grpcServer *grpc.Server) {
 }
 
 func (s *MockService) NewHTTPMux(ctx context.Context, conn *grpc.ClientConn) (http.Handler, error) {
-	mux := runtime.NewServeMux()
-
-	if err := pb.RegisterCertificateManagerHandler(ctx, mux, conn); err != nil {
+	mux, err := httpmux.NewServeMux(ctx, conn, httpmux.Options{},
+		pb.RegisterCertificateManagerHandler,
+		s.operations.RegisterOperationsPath("/v1/{prefix=**}/operations/{name}"),
+	)
+	if err != nil {
 		return nil, err
 	}
-
+	mux.RewriteError = func(ctx context.Context, error *httpmux.ErrorResponse) {
+		if error.Code == 404 {
+			switch resource := strings.Split(error.Message, " ")[0]; resource {
```

Comment:
Nit: I've started handling these in the Get<Foo> methods, personally.  But this works too!